HW3/src/js/task2.js

```diff
@@ -0,0 +1,22 @@
+let a = parseFloat(prompt('Введіть довжину сторони А'));
+let b = parseFloat(prompt('Введіть довжину сторони Б'));
+let c = parseFloat(prompt('Ввдедіть значення кута'));
+
+function lengthB(a, b, c) {
```

Comment:
please use a better name for the function parameters

Reply:
+